eng/pipelines/coreclr/templates/build-perf-sample-apps.yml

```diff
@@ -0,0 +1,45 @@
+parameters:
+  osGroup: ''
+  osSubgroup: ''
+  archType: ''
+  buildConfig: ''
+  runtimeFlavor: ''
+  helixQueues: ''
+  targetRid: ''
+  nameSuffix: ''
+  platform: ''
+  shouldContinueOnError: ''
+  rootFolder: ''
+  includeRootFolder: ''
+  displayName: ''
+  artifactName: ''
+  archiveExtension: ''
+  archiveType: ''
+  tarCompression: ''
+
+steps:
+# Build Android sample app
+  - ${{ if eq(parameters.osGroup, 'Android') }}:
+    - script: make run MONO_ARCH=arm64 DEPLOY_AND_RUN=false
```

Comment:
If you switch over to the functional tests, you can simply build them via:

`./dotnet.sh build src/tests/FunctionalTests/Android/Device_Emulator/JIT /p:TargetOS=Android /p:TargetArchitecture=<arch> /p:Configuration=Release`